components/ad/smartbanner/src/index.scss

```diff
@@ -51,11 +51,15 @@
     }
   }
 
-  &-visible {
+  &--visible {
     margin-top: 70px;
   }
 
-  &-hidden {
+  &--hidden {
     margin-top: 0;
   }
```

Comment:
Do you think this is a component responsability? i mean, looks like this class should be added somewhere outside the component.